.gitignore

```diff
@@ -24,3 +24,4 @@ genesis.dat
 *.sqlite3
 *.sqlite3-shm
 *.sqlite3-wal
+db/
```

Comment:
we should either track the directory, make the CLI create it or mention somewhere in the documentation that the user needs to create a `db` directory. Otherwise you get an error because the path doesn't exist

Reply:
Indeed. Will revert back the config file so it automatically creates it at the root.